shell/common/rasterizer.h

```diff
@@ -230,35 +230,27 @@ class Rasterizer final : public SnapshotDelegate {
   flutter::TextureRegistry* GetTextureRegistry();
 
   //----------------------------------------------------------------------------
-  /// @brief      Takes the next item from the layer tree pipeline and executes
-  ///             the raster thread frame workload for that pipeline item to
-  ///             render a frame on the on-screen surface.
+  /// @brief      Takes the latest item from the layer tree holder and executes
+  ///             the raster thread frame workload for that item to render a
+  ///             frame on the on-screen surface.
   ///
-  ///             Why does the draw call take a layer tree pipeline and not the
+  ///             Why does the draw call take a layer tree holder and not the
   ///             layer tree directly?
   ///
-  ///             The pipeline is the way book-keeping of frame workloads
-  ///             distributed across the multiple threads is managed. The
-  ///             rasterizer deals with the pipelines directly (instead of layer
-  ///             trees which is what it actually renders) because the pipeline
-  ///             consumer's workload must be accounted for within the pipeline
-  ///             itself. If the rasterizer took the layer tree directly, it
-  ///             would have to be taken out of the pipeline. That would signal
-  ///             the end of the frame workload and the pipeline would be ready
-  ///             for new frames. But the last frame has not been rendered by
-  ///             the frame yet! On the other hand, the pipeline must own the
-  ///             layer tree it renders because it keeps a reference to the last
-  ///             layer tree around till a new frame is rendered. So a simple
-  ///             reference wont work either. The `Rasterizer::DoDraw` method
-  ///             actually performs the GPU operations within the layer tree
-  ///             pipeline.
+  ///             The layer tree holder is a thread safe way to produce frame
+  ///             workloads from the UI thread and raster them on the raster
+  ///             thread. To account for scenarious where the UI thread
+  ///             continues to produce the frames while a raster task is queued,
+  ///             `Rasterizer::DoDraw` that gets executed on the raster thread,
```

Comment:
Unnecessary comma at the end of this line separates the noun from its verb, should be "... on the raster thread must pick up ..."